<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WdgActionSearch</class>
 <widget class="QWidget" name="WdgActionSearch">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QListView" name="lstAction"/>
   </item>
   <item>
    <widget class="QLabel" name="lblWhatsThis">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="bnTrigger">
     <property name="text">
      <string>E&amp;xecute Action</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
